src/components/search-wizard/Wizard.jsx

```diff
@@ -81,15 +88,48 @@ export default function SearchWizard() {
                 }`
               : null}
           </Button>
-        )}
+        ) : null}
+        {state.matches('result') ? (
+          <>
+            <Button
+              color={Button.Colors.primary}
+              appearance={Button.Appearances.solid}
+              className="w-full"
+              size={Button.Sizes.xl}
+              onClick={() => send('DOWNLOAD')}
+            >
+              Download
+            </Button>
+            <Button
+              color={Button.Colors.primary}
+              className="w-full"
+              size={Button.Sizes.xl}
+              onClick={() => send('QUERY_LAYERS')}
+            >
+              Back
+            </Button>
+          </>
+        ) : null}
+        {state.matches('download') ? (
+          <>
+            <Button
+              color={Button.Colors.primary}
+              className="w-full"
+              size={Button.Sizes.xl}
```

Comment:
**thought**: would all these ternary statements be simplified if you moved all of this to a parent component that takes the current state as a prop?

Reply:
I'm going to punt on this until I have the advanced filter working. Then I plan on taking a fresh look at this. There's got to be a better way.